src/app/Main/Body/QueryPage/NumberOfIsolates/QueryPage-NrOfIsolates.component.tsx

```diff
@@ -0,0 +1,67 @@
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+import { useTranslation } from "react-i18next";
+import { useContext } from "react";
+import Accordion from "@material-ui/core/Accordion";
+import AccordionSummary from "@material-ui/core/AccordionSummary";
+import AccordionDetails from "@material-ui/core/AccordionDetails";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { DataContext } from "../../../../Shared/Context/DataContext";
+import { countNrOfIsolates } from "./countNrOfIsolates.component";
+
+const subHeadingStyle = css`
+    margin: 0;
+`;
+const parameterBlockStyle = css`
+    margin-left: 2em;
+`;
+const tableStyle = css`
+    table-layout: auto;
+`;
+const tableTextStyle = css`
+    padding-right: 1em;
+    font-weight: bold;
+`;
+const tableNumberStyle = css`
+    padding-right: 5em;
+`;
+
+export function QueryPageNrOfIsolatesComponent(): JSX.Element {
+    const { t } = useTranslation(["QueryPage"]);
+    const { data } = useContext(DataContext);
+
+    const totalNrOfIsolates = countNrOfIsolates(data.ZNData);
+    const nrOfSelectedIsolates = countNrOfIsolates(data.ZNDataFiltered);
+
+    return (
+        <Accordion defaultExpanded>
+            <AccordionSummary
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls="panel1a-content"
+                id="panel1a-header"
+            >
+                <h3 css={subHeadingStyle}>{t("NrOfIsolates.Title")}</h3>
+            </AccordionSummary>
+            <AccordionDetails css={parameterBlockStyle}>
+                <table css={tableStyle}>
+                    <tbody>
+                        <tr>
+                            <td key="tableText-total" css={tableTextStyle}>

```

Comment:
whats that key attribute?

Reply:
Each table element needs a unique key, otherwise there will be a browser warning

Reply:
I can not find any documentation about the key attribute. Is this a feature of some library you use? Whats the exact message of the warning?

Reply:
The warning was "Warning: Each child in a list should have a unique "key" prop". 
But it seems that you are right and I need the key only for lists (material-ui) and not for tables. I don't remember why I used it here.

Reply:
Ok this is a react attribute:
https://reactjs.org/docs/lists-and-keys.html
There is a similar feature in our angular apps. It is for helping change detection to identifying unchanged elements. E.g. if you reorder a list react do not need to render the individual items because they are unchanged but only change the layout. I would consider reading into this further, there seems to be some misunderstanding how and when to use this feature and false usage can lead to bugs during rendering.